### PR TITLE
CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by stokito**
Little bit updated the CONTRIBUTING.md file

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ Pull requests are the easiest way to contribute changes to git repos at Github.
 
 ## Release Branches:
 
-* Branches named "for-XX.YY" or "luci-X.Y" (e.g. "for-15.05") are release branches.
+* Branches named `for-XX.YY` or `luci-X.Y` (e.g. `for-15.05`) are release branches.
```
Comment (thess) [inline CONTRIBUTING.md]: Going forward, I believe branch names will be in the form: `openwrt-xx.yy` (Ex: `openwrt-18.06` is current release branch.)

**Comment by thess** [comment]
Thanks for the update - see comments

File: CONTRIBUTING.md (hunk 1 above)
Comment (stokito) [reply CONTRIBUTING.md]: so, should I update it?

Comment (thess) [reply CONTRIBUTING.md]: That was the idea.

**Pull request merged**